forkpi/records/models.py

```diff
@@ -61,3 +61,21 @@ class Option(Model):
 class Door(Model):
     name = TextField(default="", unique=True)
     serial = TextField(unique=True)
+
+
+class AppConfigManager(Manager):
+    def get_singleton(self):
```

Comment:
```suggestion
    def get_or_create_singleton(self, global_pin=None):
```

Why not change the function to this? You can then get rid of `update_global_pin`.